script.js

```diff
@@ -0,0 +1,97 @@
+const API_KEY = 'b6ad49d4';
+const RESULTS_PER_PAGE = 10;
```

Comment:
And here - better to hide completely the `div` on the right side. Because at the beginning it seems that something is missing

<img width="923" alt="Screenshot 2022-11-18 at 11 43 04" src="https://user-images.githubusercontent.com/34886027/202671414-81c8bd18-ff94-473d-9be5-7705aa2cd504.png">